page-campaigns.php

```diff
@@ -1,6 +1,8 @@
 <?php 
-    get_header();
-    $p = intval(get_query_var('a')) <= 1 ? 1 : intval(get_query_var('a'));
+	get_header();
+	$user = wp_get_current_user();
+	$subscribed = get_post_meta($user->ID, 'newsletter', true) ? get_post_meta($user->ID, 'newsletter', true) : '';
```

Comment:
Shouldn't need to call get_post_meta twice here.  Store it in a variable first then do the check, this ends up being two database calls.

Also for user meta data there is the function get_user_meta

https://developer.wordpress.org/reference/functions/get_user_meta/